### DISCUSSION TITLE
Are firestore bindings supposed to update, or only read?

### DISCUSSION THREAD

**Discussion by ldiebold** [vueuse/vueuse#874] [Q&A]
I've been using `useFirestore` to keep data in sync and am a little confused :sweat_smile: 

When I update data in the UI, it changes on the frontend (as expected)

However, updating the ref on the frontend **does not update data in the cloud**. Is this the expected behaviour?

If so, perhaps the docs need to be clearer? The following implies to me that updates to the ref will also update data in the cloud...

> Making it straightforward to always keep your local data in sync with remotes databases.

Let me know if a code example is needed, or if I simply misunderstood the docs :smile:

**Comment by wheatjs** [comment]
I don't believe it will actually set data by updating a ref. Maybe we can update the types for the Ref to be readonly.

**Reply by ldiebold** [reply]
Okay, thankyou!

There's still some potential benefit of allowing it to change. e.g. optimistic UIs perhaps.

**Reply by wheatjs** [reply]
Perhaps we could have an option in `useFirestore` that will enable automatically updating a firestore doc when you update a ref
